include/mbedtls/config.h

```diff
@@ -1909,16 +1909,14 @@
 #define MBEDTLS_VERSION_FEATURES
 
 /**
- * \def MBEDTLS_X509_ALLOW_UNSUPPORTED_CRITICAL_EXTENSION
+ * \def MBEDTLS_X509_ALLOW_EXTENSIONS_NON_V3
```

Comment:
This option was removed in #4581, we don't want to re-introduce it here. This is probably an issue with conflict resolution in the merge commit.